.github/workflows/linter.yml

```diff
@@ -40,10 +40,14 @@ jobs:
         id: super-linter
         uses: super-linter/super-linter/slim@v6
         env:
-          DEFAULT_BRANCH: main
-          FILTER_REGEX_EXCLUDE: dist/**/*
+          DEFAULT_BRANCH: master
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          FILTER_REGEX_EXCLUDE: (dist|assets)/**/*
           TYPESCRIPT_DEFAULT_STYLE: prettier
+          IGNORE_GITIGNORED_FILES: true
           VALIDATE_ALL_CODEBASE: true
-          VALIDATE_JAVASCRIPT_STANDARD: false
-          VALIDATE_JSCPD: false
+          VALIDATE_JAVASCRIPT_STANDARD: true
```

Comment:
1. please use `VALIDATE_JAVASCRIPT_ES` instead, to utilize `eslint`.
2. `VALIDATE_TYPESCRIPT_ES` is more important as most of the code is TS and not JS, but you didn't configure it. Please add.

Reply:
3. please add `VALIDATE_TSX` for react

Reply:
VALIDATE_JAVASCRIPT_ES,  VALIDATE_TYPESCRIPT_ES and VALIDATE_TSX default value are true

Reply:
1. I didn't see them running
2. `VALIDATE_JAVASCRIPT_ES` instead of `VALIDATE_JAVASCRIPT_STANDARD`

Reply:
I added, there is an issue with the linter working with the new config format for now, independent from me, notify your-self on progress [here](https://github.com/super-linter/super-linter/issues/5688#issue-2317198126)